ITR/data/base_providers.py

```diff
@@ -285,23 +319,16 @@ def _calculate_target_projections(self, production_bm: BaseProviderProductionBen
             else:
                 base_year_production = next((p.value for p in c.historic_data.productions if
                                              p.year == self.temp_config.CONTROLS_CONFIG.base_year), None)
-                with warnings.catch_warnings():
-                    warnings.simplefilter("ignore")
-                    company_sector_region_info = pd.DataFrame({
```

Comment:
why don't we need this dataframe any more?

Reply:
Because I realized how inefficient it was to round-trip a complete re-derivation of data that's actually in our hands.  df_partial_pp has computed everything we could want, and it's done so outside the loop.  Recomputing it inside the loop is pure waste.